src/Gaufrette/Util/Path.php

```diff
@@ -86,4 +86,39 @@ public static function dirname($path)
     {
         return str_replace('\\', '/', \dirname($path));
     }
+
+    /**
+     * UTF-8 aware parse_url() replacement.
+     *
+     * @param string $url to parse
+     *
+     * @return bool|array
+     *
+     * @see https://secure.php.net/manual/fr/function.parse-url.php#114817
```

Comment:
Can you change this link to use the english locale please ? (`manual/fr` => `manual/en`) : https://secure.php.net/manual/en/function.parse-url.php#114817